### DISCUSSION TITLE
Exclude current domain from pop up menu

### DISCUSSION THREAD

**Discussion by baka24** [violentmonkey/violentmonkey#1748] [Q&A] [answered]
In Tampermonkey, I can easily exclude script for current domain

<details> 
  <summary>Tampermonkey</summary>

![NkptGkE7pL](https://user-images.githubusercontent.com/15723214/218307329-bc56c239-b490-4631-bfff-b539b2f2dff8.png)

</details>

Exclude in Violentmonkey is from current URLs instead from current domain

<details> 
  <summary>Violentmonkey</summary>

![0oYcHKkwls](https://user-images.githubusercontent.com/15723214/218307418-8d8f32bc-9bdd-426a-8deb-b31a6e5e81f9.png)

</details>

Can I have same exclude function like TM in VM?

**Comment by tophf** [comment, marked as answer]
You can replace the portion after the domain with `*` e.g. in your example it'll be `https://www.bing.com/*`. This info is displayed when you click the (i) icon, which was recently broken though, but it'll be restored in the next release:

![image](https://user-images.githubusercontent.com/1310400/218308410-3717cbf9-fe2e-4476-8eaa-89743e11d523.png)

I guess we can improve the UI:
1. show this info by default and remember its state next time if it was collapsed
2. allow clicking on the choices (maybe show them as buttons) and change the value in the prompt accordingly
   ![image](https://user-images.githubusercontent.com/1310400/218315612-8d57e7ab-95f9-4615-8367-6ae2fe0f0872.png)

See #1749.

**Reply by baka24** [reply]
Ah Thanks, I like the new pop-up :)
